src/workerd/io/trace.c++

```diff
@@ -1035,8 +1037,10 @@ kj::Maybe<tracing::SpanOpen::Info> readSpanOpenInfo(rpc::Trace::SpanOpen::Reader
 }  // namespace
 
 tracing::SpanOpen::SpanOpen(rpc::Trace::SpanOpen::Reader reader)
+    // TODO(streaming-tail): Propagate parentSpanId properly
     : operationName(readSpanOpenOperationName(reader)),
-      info(readSpanOpenInfo(reader)) {}
+      info(readSpanOpenInfo(reader)),
+      parentSpanId(0) {}
```

Comment:
Shouldn't this be populated from the reader?

Reply:
In practice, yes – there is no parent Span ID tag defined for the SpanOpen reader so far though. We certainly need to define this somewhere in the capnp representation eventually, but this PR is mostly intended to unblock evaluation, so I just wanted to make a parent span ID available in the common case. Where exactly the parent Span ID should be defined is still something we need to finalize.